Core/include/Acts/Vertexing/GaussianTrackDensity.ipp

```diff
@@ -53,8 +53,10 @@ Acts::GaussianTrackDensity<input_track_t>::globalMaximumWithWidth(
                       maxDensity, maxSecondDerivative);
   }
 
-  return std::make_pair(maxPosition,
-                        std::sqrt(-(maxDensity / maxSecondDerivative)));
+  return maxSecondDerivative == 0.
```

Comment:
```suggestion
  return (maxSecondDerivative == 0.)
```
I prefer to group the boolean expression with extra brackets to avoid any confusion.

Reply:
Done.